pipelines/LSSTCam-imSim/ApVerify.yaml

```diff
@@ -15,6 +15,12 @@ tasks:
     class: lsst.ap.association.DiaPipelineTask
     config:
       doPackageAlerts: True
+  rbClassify:
+    class: lsst.meas.transiNet.TransiNetTask
+    config:
+      modelFile: testModel
+      weightsFile: '/home/nima/meas_transiNet/tests/data/checkpoint.pth.zip'
```

Comment:
I don't think this will work -- `ap_verify` datasets must be completely self-contained (in particular, this path won't be visible from Jenkins). Is this "weights file" something that can be added to the Butler repository, like solar system ephemerides were for the HiTS and COSMOS datasets?

Reply:
This is the discussion I was pointing at during today's meeting. We are aware of the need to ingest these big chunks of data into the repository. But the question is how. There is https://jira.lsstcorp.org/browse/DM-36742 where I think your comments are needed.
However, for making use of the real-bogus classifier during the upcoming sprint, I think we can still use it in its current hacky, machine-dependent form.

As for whether it is instrument- or data-specific, I lean towards the former. It should NOT depend on the test data -- but we will have various versions, and the user should be able to specify them when defining the pipeline, which is another discussion to happen in the above ticket I guess.

Reply:
If the model is instrument-wide then it probably doesn't belong in a repository; I've said as much on DM-36742 (thanks for pointing me to it!). Putting it in one of the obs packages would also solve the present issue of having a platform-dependent path, since then the path would be relative to a package directory (which is required to be set up by the dataset definition).

However, I disagree that this is an acceptable hack in the meantime, because it will break our nightly builds indefinitely. I'd recommend making this issue blocked on DM-36742 and using a local pipeline for the sprint, or at least putting the `TransiNetTask` pipeline into a separate pipeline that isn't in any of the standard execution paths (i.e., not `<dataset>/ApVerify.yaml` or `<dataset>/ApVerifyWithFakes.yaml`, nor any of the pipelines they include).

Reply:
Given the implementation of model packages and the current git-repository-based backend, I see this discussion as resolved. Please feel free to reopen it in case I'm mistaken.